scala/cz-tictactoe.scala

```diff
@@ -83,15 +98,21 @@ class Board() {
 
   var fields = Array.tabulate(3,3)( (x,y) => UNOCCUPIED )
 
-  // FIXME: change to `override def toString(): String` and let `Game` draw it
-  def printField(): Unit = {
+  override def toString(): String = {
+    var board = 
+      "  X → 1  2  3\n" + 
+      "Y  \n" + 
+      "↓\n" + 
+      "\n"
+
     for(i <- 0 until 3) {
+      board += ((i+1) + "    ")
       for(j <- 0 until 3) {
-        print(" " + fields(j)(i) + " ")
+        board += (" " + fields(j)(i) + " ")
       }
-      println
-      println
+      board += "\n\n"
```

Comment:
Would probably remove the two CR/LFs and let that be handled by the caller

Reply:
Hm, that does not work, we need the newline breaks here!